### PR TITLE
KIALI-2918 Enable group nodes to have a context menu

### PR DESCRIPTION
**Pull request by josejulio**
** Describe the change **

- Moved some redux properties to redux-connect syntax
- In versioned graph, the versioned node (v1, v2) will be considered as workloads
- Droped the version from the title because it won't be valid to show a workload name with a version
- NodeType=unknown and isInaccessible won't show any menu

** Screenshot **
Peek 2019-05-31 11-42

### CODE REVIEW

**Comment by jshaughn** [comment]
@josejulio I think now that the other work is in you can update this PR appropriately and probably use it to make the adjustments you recommended in [The 1209 PR](https://github.com/kiali/kiali-ui/pull/1209).

File: src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx [hunk 1]
```diff
@@ -72,27 +83,35 @@ export class NodeContextMenu extends React.PureComponent<NodeContextMenuProps, N
   }
 
   render() {
-    const version = this.props.version !== '' ? `:${this.props.version}` : '';
-    const detailsPageUrl = this.makeDetailsPageUrl();
-    const { nodeType, app } = this.state;
+    // Disable context menu if we are dealing with a unknown node
```
Comment (jshaughn) [inline src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx]: We should also disable content menu for isInaccessible nodes.  These nodes are not actually accessible to the user, they are rendered only to show a connection and are typically in a restricted namespace or may be service entries.

**Comment by josejulio** [comment]
> @josejulio Looks good, just one comment to take deal with before approval. Also, in the video there are errors popping up, is that anything to be concerned with? 

Not really, these errors were related to a call my server didn't had at that poin.

> One last note, when you make the videos maybe move a little more slowly because the mouse movement often does not show up and it's hard to follow (at least for me).

My bad, I disabled the mouse cursor from appearing in the video. I'll record again with mouse on.

File: src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx (hunk 1 above)
Comment (josejulio) [reply src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx]: Sounds good

Comment (josejulio) [reply src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx]: Done

**Comment by josejulio** [comment]
@jshaughn updated the gif, hope is better now.
